moj-node/server/views/components/search/template.njk

```diff
@@ -0,0 +1,9 @@
+<div  id="search-wrapper">
+  <form method="GET" action="/search">
+    <label class="govuk-label" for="event-name">Search</label>
+    <input class="govuk-search-input search-box" id="search" name="query" type="text" value="{{params.query}}" autocomplete="off" placeholder="Search the content hub">
```

Comment:
Is the placeholder text Nigel approved 😅 ?

Reply:
Yes